cli/pack/systemd_dir.go

```diff
@@ -161,6 +163,7 @@ Environment=TARANTOOL_WORKDIR={{ .DefaultWorkDir }}
 Environment=TARANTOOL_CFG={{ .ConfPath }}
 Environment=TARANTOOL_PID_FILE={{ .DefaultPidFile }}
 Environment=TARANTOOL_CONSOLE_SOCK={{ .DefaultConsoleSock }}
+Environment=TARANTOOL_NET_MSG_MAX={{ .NetMsgMax }}
```

Comment:
```
tarantool> box.cfg{net_msg_max = -2}
...
2021-06-22 11:52:22.297 [39038] main/103/interactive F> can't initialize storage: Incorrect value for option 'net_msg_max': minimal value is 2
```

Are there any check to prevent such behaviour? In case if you write invalid value to unit file it won't be possible even start such service.